extensions/AzureCosmosDBMongoVCore/AzureCosmosDBMongoVCore/AzureCosmosDBMongoVCoreConfig.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Runtime.Serialization;
+
+namespace Microsoft.KernelMemory;
+
+/// <summary>
+/// Get more details about Azure Cosmos Mongo vCore and these configs https://learn.microsoft.com/en-us/azure/cosmos-db/mongodb/vcore/vector-search
```

Comment:
MongoDB. I'll leave to you to find all the places :-)

Reply:
Updated the name